src/System.Net.WebSockets/tests/WebSocketExceptionTests.cs

```diff
@@ -179,12 +179,5 @@ public void ConstructorTests_Message_Exception_Success()
             Assert.Equal(Message, wse.Message);
             Assert.Same(inner, wse.InnerException);
         }
-
-        [Fact]
```

Comment:
This is no bug to track because this type does not support serialization and there is no plan in .NET Core for it to support it.